Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/SANSILLIntegration.py

```diff
@@ -329,11 +329,19 @@ def _setup_mildner_carpenter(self):
             y3 = instrument.getNumberParameter('y-pixel-size')[0] / 1000
         else:
             raise RuntimeError('Unable to calculate resolution, missing pixel size.')
-        delta_wavelength = run.getLogData('selector.wavelength_res').value * 0.01
+        try:
```

Comment:
run object has a method .hasLogData(), using try-except is an overkill here. 
run is also a python dictionary, one could even do:
`if log in run:`

Reply:
maybe I confused with hasParameter of instrument, anyway, if it's there it should be in API documentation